submarine-workbench/workbench-server/src/main/java/org/apache/submarine/database/service/TeamMemberService.java

```diff
@@ -46,39 +46,4 @@
     }
     return list;
   }
-
-  public void add(TeamMember teamMember) throws Exception {
```

Comment:
Why do you delete `add()` and `deleteByprimaryKey()`?

Reply:
These two methods are not used in the current business to prevent excessive redundant code, so I deleted them first.